### PR TITLE
Button component

### PR DESCRIPTION
**Pull request by tinenhao**
image
image
image
Images from Trello
image

### CODE REVIEW

File: frontend/src/components/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Border = styled.button<{ state: string }>`
+  height: 2.2rem;
+  width: 10vw;
```
Comment (kiyonggg) [inline frontend/src/components/Button.tsx:6]: if the button text is too long, the text will be cut off

File: frontend/src/components/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+`
+const Inner = styled.div<{ state: string }>`
+  height: 100%;
+  width: 100%;
+  border-radius: 2rem;
+  padding-top: 0.35rem;
+  background: ${(props) => (props.state === 'primary' ? `linear-gradient(to left, #72BA75, #58B994)` : 'white')};
+  color: ${(props) => (props.state === 'primary' ? `white` : props.state === 'secondary' ? `#58B994` : '#BFBFBF')};
+`
+type Props = {
```
Comment (kiyonggg) [inline frontend/src/components/Button.tsx:24]: props might need onClick function?

Comment (tinenhao) [reply frontend/src/components/Button.tsx:24]: Fixed in 6ec062d

File: frontend/src/components/Button.tsx (hunk 1 above)
Comment (tinenhao) [reply frontend/src/components/Button.tsx:6]: Communicated with designers and will leave it as it is first since likelihood of this happening is very low

**Pull request merged**